libatlas-base-dev has no package

### Linux Distribution

Debian 13 (trixie) 

### What happened

Installing Input Shaper extra dependencies throws an error

`[ERROR] Error installing packages:
E: Package 'libatlas-base-dev' has no installation candidate`

### What did you expect to happen

Should install an alternative:
[https://github.com/numpy/numpy/issues/29108](url)

### How to reproduce

Install input shaper extra dependencies from advanced section. 

### Additional information

_No response_

--- a/kiauh/components/klipper/klipper_utils.py
+++ b/kiauh/components/klipper/klipper_utils.py
@@ -237,7 +237,6 @@ def install_input_shaper_deps() -> None:
             "If you agree, the following additional system packages will be installed:",
             "● python3-numpy",
             "● python3-matplotlib",
-            "● libatlas-base-dev",
             "● libopenblas-dev",
             "\n\n",
             "Also, the following Python package will be installed:",
@@ -253,7 +252,6 @@ def install_input_shaper_deps() -> None:
     apt_deps = (
         "python3-numpy",
         "python3-matplotlib",
-        "libatlas-base-dev",
         "libopenblas-dev",
     )
     check_install_dependencies({*apt_deps})

